client/src/components/History/ContentOperations.vue

```diff
@@ -31,6 +31,7 @@
                     size="sm"
                     text="Selection"
                     :disabled="!hasSelection"
```

Comment:
Would it be more logical to hide it completely, instead of cluttering up interface with disabled button?
if nothing is selected, there is no much use to it in, I suppose 
```suggestion
                    :v-if="!hasSelection"
```

Reply:
I think it actually serves a useful purpose to saying "Hey, if you select things, there's more stuff you can do", for this one.

Reply:
> I think it actually serves a useful purpose to saying "Hey, if you select things, there's more stuff you can do", for this one.

well, I don't have a strong opinion on it. I think it's better, but we can leave it if you feel like it :+1: